srcpkgs/libsigsegv/template

```diff
@@ -1,14 +1,14 @@
 # Template file for 'libsigsegv'
 pkgname=libsigsegv
-version=2.12
+version=2.13
 revision=1
 build_style=gnu-configure
 short_desc="Handling page faults in user mode"
 maintainer="Martin Riese <grauehaare@gmx.de>"
 license="GPL-2"
 homepage="https://www.gnu.org/software/libsigsegv/"
 distfiles="https://ftpmirror.gnu.org/libsigsegv/libsigsegv-${version}.tar.gz"
-checksum=3ae1af359eebaa4ffc5896a1aee3568c052c99879316a1ab57f8fe1789c390b6
+checksum=be78ee4176b05f7c75ff03298d84874db90f4b6c9d5503f0da1226b3a3c48119
 configure_args="--enable-shared"
 
 post_configure() {
```

Comment:
Please move to post_patch and use vsed.

Reply:
Or possibly just try to remove this block and see if it builds.

Reply:
config.h is created by configure.